src/libraries/System.Private.CoreLib/src/System/IO/FileSystem.Windows.cs

```diff
@@ -556,8 +556,8 @@ internal static void CreateSymbolicLink(string path, string pathToTarget, bool i
             // The file or directory is not a reparse point.
             if ((data.dwFileAttributes & (uint)FileAttributes.ReparsePoint) == 0 ||
                 // Only symbolic links and mount points are supported at the moment.
-                ((data.dwReserved0 & Interop.Kernel32.IOReparseOptions.IO_REPARSE_TAG_SYMLINK) == 0 &&
-                 (data.dwReserved0 & Interop.Kernel32.IOReparseOptions.IO_REPARSE_TAG_MOUNT_POINT) == 0))
+                (data.dwReserved0 != Interop.Kernel32.IOReparseOptions.IO_REPARSE_TAG_SYMLINK &&
```

Comment:
do the above tests verify BOTH of these operators?

Reply:
Yes, the test `UnsupportedLink_ReturnsNull` uses a ReparsePoint with a ReparseTag of IO_REPARSE_TAG_APPEXECLINK so both conditions are checked.